libs/datasets/data_version.py

```diff
@@ -1,104 +0,0 @@
-import click
```

Comment:
This code works with the covid-data-public repo, but it is never used anywhere, so I've removed it here